modules/migration/hooks/useMigrationStatus.tsx

```diff
@@ -2,12 +2,18 @@ import { useDelegatedTo } from 'modules/delegates/hooks/useDelegatedTo';
 import { useActiveWeb3React } from 'modules/web3/hooks/useActiveWeb3React';
 import { useDelegateContractExpirationDate } from 'modules/delegates/hooks/useDelegateContractExpirationDate';
 import { isAboutToExpireCheck, isExpiredCheck } from '../helpers/expirationChecks';
+import { addressConnections } from 'modules/migration/connections';
 
-export function useDelegationMigrationStatus(): {
+export function useMigrationStatus(): {
   isDelegatedToExpiredContract: boolean;
   isDelegatedToExpiringContract: boolean;
   isDelegateContractExpired: boolean;
   isDelegateContractExpiring: boolean;
+  newOwnerAddress?: string;
+  newOwnerConnected: boolean;
+  previousOwnerAddress?: string;
+  previousOwnerConnected: boolean;
+  newOwnerHasDelegateContract: boolean;
```

Comment:
can we separate this into another hook for delegates? because this hook is used by delegators and delegates and it feels like a mix of things now. 

I think the first 4 variables have enough entity to be considered outside the migration status

Reply:
We can separate them but I'm not sure how you are proposing to separate them? Which ones to include and which to separate?